src/test/testgear.cc

```diff
@@ -239,12 +239,12 @@ void testFixedPadSizeDiskLayout( const  FixedPadSizeDiskLayout& pl ) {
       // in the other implementations, because there are pads at the edge of the pad plane
       // which have no neighbour.
       if( j == 0 ) {
-	int ln = pl.getRightNeighbour(  pl.getPadIndex( iRow , iPad ) ) ; 
+	pl.getRightNeighbour(  pl.getPadIndex( iRow , iPad ) ) ; 
```

Comment:
```suggestion
    [[maybe_unused]] int ln = pl.getRightNeighbour(  pl.getPadIndex( iRow , iPad ) ) ;
```
There is an `assert` just below that will fail depending on the build type potentially. (Few more below)

Reply:
Hmm, it's weird that it compiled fine

Reply:
I think it depends on the compiler. gcc seems to have some sort of `#define assert(x)` that is effectively empty, where as clang was complaining about undefined symbols. Potentially also depending on the optimization level.